src/test/java/bio/overture/ego/service/FirstUserAsAdminTest.java

```diff
@@ -0,0 +1,43 @@
+package bio.overture.ego.service;
+
+import bio.overture.ego.model.entity.User;
+import bio.overture.ego.model.enums.UserType;
+import bio.overture.ego.utils.EntityGenerator;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+@Slf4j
+@SpringBootTest
+@RunWith(SpringRunner.class)
+@ActiveProfiles("test")
+@TestPropertySource(properties = "default.user.firstUserAsAdmin=true")
+@Transactional
+public class FirstUserAsAdminTest {
+
+  @Autowired private UserService userService;
+  @Autowired private EntityGenerator entityGenerator;
+
+  @Test
+  public void testOnlyFirstUserShouldBeAdminByDefault() {
```

Comment:
Can we convert these tests to controller tests? Rob has indicated that these are more reliable when it comes to hibernate weirdness: https://github.com/overture-stack/ego/pull/488#pullrequestreview-463423170 
Maybe in the `CreateUserControllerTest`, to take advantage of the idtoken mock?

Reply:
I'll try to add another test there, but I don't want to spend more time than needed for this ticket, it's too small anyway. 

unless there is a clear reason why this is not sufficient to catch regressions I don't think we should do controller level tests for everything, they come at a cost

Reply:
Adding controller test opened a can of worms because using @TestPropertySource breaks spring context between tests:

so TestSourceProperty is problematic for the controller tests not sure how that collides, but spring always does nasty shit when it comes to tests and cache context and what not, so based on the time I already wasted on this I'll remove the controller test